pkg/controllers/management/auth/crtb_handler.go

```diff
@@ -205,12 +218,16 @@ func (c *crtbLifecycle) removeMGMTClusterScopedPrivilegesInProjectNamespace(bind
 	return nil
 }
 
+// reconcileLabels updates labels for the provided CRTB.
+//
+// Prior to 2.5, for every CRTB, the following CRBs and RBs are created in the management clusters
```

Comment:
As mentioned here: https://github.com/rancher/rancher/pull/42473/files#r1339217868
This should not be moved out of the body